pyaedt/generic/configurations.py

```diff
@@ -1700,22 +1700,27 @@ def _export_mesh_operations(self, dict_out):
         dict_out["mesh"]["Settings"] = mop["Settings"]
         if self._app.mesh.meshregions:
             for mesh in self._app.mesh.meshregions:
-                if mesh.name == "Settings":
+                if mesh.name in ["Settings", "Global"]:
                     args = ["NAME:Settings"]
                 else:
                     args = ["NAME:" + mesh.name, "Enable:=", mesh.Enable]
                 args += mesh.settings.parse_settings_as_args()
-                args += getattr(mesh, "_parse_assignment_value")()
+                if mesh.name not in ["Settings", "Global"]:
+                    args += getattr(mesh, "_parse_assignment_value")()
```

Comment:
Can you move that line in the previous `else` statement ? You would be in the case were `mesh.name not in ["Settings", "Global"]`. Unless you need this to be added after `mesh.settings.parse_settings_as_args()`